docs/specifications/tests/Delegation-TP/delegation01.md

```diff
@@ -1,48 +1,119 @@
+and IPv6 addresses respectively.
+
+
+## Inputs
+
+The domain name to be tested ("child zone").
+
+## Ordered description of steps to be taken to execute the test case
+
+1. Obtain the complete set of the names of the name servers for the *child 
+   zone* (distinct NS records) from parent using [Method2].
+2. If the number of name servers from parent is at least two, emit the
+   *ENOUGH_NS_PARENT*, else emit *NOT_ENOUGH_NS_PARENT*.
+3. Obtain the IP addresses for the set of the name servers from parent 
+   using [Method4].
+4. If there are no name servers from parent resolving to an IPv4 
+   address then emit *NO_IPV4_NS_PARENT*.
+5. If the number of name servers from parent resolving to an IPv4 
+   address is one then emit *NOT_ENOUGH_IPV4_NS_PARENT*.
+6. If the number of name servers from parent resolving to an IPv4 
+   address is at least two then emit *ENOUGH_IPV4_NS_PARENT*.
+7. If there are no name servers from parent resolving to an IPv6 
+   address then emit *NO_IPV6_NS_PARENT*.
+8. If the number of name servers from parent resolving to an IPv6 
+   address is one then emit *NOT_ENOUGH_IPV6_NS_PARENT*.
+9. If the number of name servers from parent resolving to an IPv6 
+   address is at least two then emit *ENOUGH_IPV6_NS_PARENT*.
+10. Obtain the complete set of the names of the name servers for the 
+    *child zone* from the *child zone* using [Method3].
+11. If the number of name servers from child is at least two emit the
+   *ENOUGH_NS_CHILD*, else emit *NOT_ENOUGH_NS_CHILD*.
+12. Obtain the IP addresses for the set of the name servers from 
+    *child zone* using [Method5] and, for 
+    out-of-bailiwick NS, using [Method4].
+13. If there are no name servers from child resolving to an IPv4 
+   address then emit *NO_IPV4_NS_CHILD*.
+14. If the number of name servers from child resolving to an IPv4 
+   address is one then emit *NOT_ENOUGH_IPV4_NS_CHILD*.
+15. If the number of name servers from child resolving to an IPv4 
+   address is two then emit *ENOUGH_IPV4_NS_CHILD*.
+16. If there are no name servers from child resolving to an IPv6 
+   address then emit *NO_IPV6_NS_CHILD*.
+17. If the number of name servers from child resolving to an IPv6 
+   address is one then emit *NOT_ENOUGH_IPV6_NS_CHILD*.
+18. If the number of name servers from child resolving to an IPv6 
+   address is two then emit *ENOUGH_IPV6_NS_CHILD*.
 
-Section 4.1 of [RFC 1034](https://tools.ietf.org/html/rfc1034) specifies that
-there must be at least minimum two name servers for a domain. This test is
-done to verify this condition
+ 
+## Outcome(s)
+
+The outcome of this Test Case is "fail" if there is at least one message
+with the severity level *ERROR* or *CRITICAL*.
+
+The outcome of this Test Case is "warning" if there is at least one message
+with the severity level *WARNING*, but no message with severity level
+*ERROR* or *CRITICAL*.
+
+In other cases the outcome of this Test Case is "pass".
```

Comment:
Would it be possible to avoid causal use of the word "case" here?

Reply:
Will update, but please note that all updated test cases probably have the same text.